straxen/plugins/x1t_cuts.py

```diff
@@ -121,39 +114,30 @@ def other_s2_bound(cls, s2_area):
 
         return rescaled_s2_0 * another_term_0 + rescaled_s2_1 * another_term_1
 
-    def compute(self, events):
+    def cut_by(self, events):
```

Comment:
Here's an old bug `events['s2_largest_other']` should now be `events['alt_s2_area']`